test/resources/openshift_api.go

```diff
@@ -0,0 +1,63 @@
+package resources
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	projectv1 "github.com/openshift/api/project/v1"
+	corev1 "k8s.io/api/core/v1"
+
+	"io/ioutil"
+	"net/http"
+)
+
+// returns all pods in a namesapce
+func DoOpenshiftGetNamespacePods(masterUrl, path, token string) (*corev1.PodList, error) {
+	resp, err := DoOpenshiftGetRequest(masterUrl, path, token)
```

Comment:
We could probably build the path endpoint dynamically, which means we'd only have to pass in the namespace to this function:
```go
path := fmt.Sprintf("/api/kubernetes/api/v1/namespaces/%s/pods", namespace)
```
Might be slightly more readable, instead of having to somewhat duplicate the api endpoint every time?